manual_flagging/initial_flagging.py

```diff
@@ -121,153 +140,375 @@
 df = df.astype(conversion_dict)
 df["ptax_flag_original"].fillna(False, inplace=True)
 
-# Separate res and condo sales based on the indicator column
-df_res = df[df["indicator"] == "res"].reset_index(drop=True)
-df_condo = df[df["indicator"] == "condo"].reset_index(drop=True)
+# Filter to correct tris
+tri_stat_groups = {
+    tri: method
+    for tri, method in inputs["tri_stat_groups"].items()
+    if tri in inputs["run_tri"]
+}
 
-# Create condo stat groups. Condos are all collapsed into a single class,
-# since there are very few 297s or 399s
-condo_stat_groups = inputs["stat_groups"].copy()
-condo_stat_groups.remove("class")
+# Handle current methodology data manipulation if needed (temporary, need to find a better way)
```

Comment:
Given that the bin partitions are different (at least currently in the `current` methodology) depending on which property type, there would have to be a conditional check somewhere no matter what since the bins would have to be specific to the filtered data. I could theoretically make two different column for `res_multi_family` and `res_single_family`, but I would still have to do a conditional check to properly allocate the columns.

You'll have to let me know if you can think of a better way to do this. One potential positive of this feature creation loop is that it is its own section where we can see for which data we are doing which extra splits, potentially making it slightly more readable? Not sure.

Reply:
I think that makes sense, I'll change it.